config/source/multi/deployments/adapter.yaml

```diff
@@ -78,7 +78,7 @@ spec:
         - name: profiling
           containerPort: 8008
 
-      terminationGracePeriodSeconds: 10
+      terminationGracePeriodSeconds: 0 # for forceful termination and restart for quick recovery
```

Comment:
It should really not be `0`. What about keeping `10` and address this quick recovery in a followup PR?